torchvision/prototype/models/quantization/resnet.py

```diff
@@ -54,7 +56,9 @@ def _resnet(
 _common_meta = {
     "size": (224, 224),
     "categories": _IMAGENET_CATEGORIES,
+    "interpolation": InterpolationMode.BILINEAR,
     "backend": "fbgemm",
+    "quantization": "ptq",
```

Comment:
Adding some additional meta-data such as interpolation, type of quantization and a reference to the unquantized enum of the the weights to all previous models.